index.html

```diff
@@ -149,19 +146,20 @@
                             or sign-ups required.</p>
                         <ul class="list-style-type-none d-flex lang weight--500">
                             <li>html</li>
+                            <li>Ruby on rails</li>
                             <li>css</li>
                             <li>javaScript</li>
                         </ul>
                         <button class="btn weight--500">See Project</button>
                     </aside>
                 </article>
             </div>
-        </section>
+        </main>
 
-        <footer>
-            <section class="container">
-                <article>
-                    <aside class="pt-114">
+        <footer class="pt-114">
```

Comment:
♻️ I appreciate how similar the styles you applied are to the template!! However, please note that some pixels before the design becomes desktop version, the about section is overlapping the last project card:

![image](https://user-images.githubusercontent.com/26911718/206576136-10212a87-ebbc-4ede-9ff1-5e20d7787032.png)

At that size, it's very likely that tablet users encounter this problem.

- [ ] Please make sure that your mobile version does not get affected when applying styles to the desktop version. Please double-check the about section/projects section when getting close to the 768px breakpoint